docs/source/pipelines.rst

```diff
@@ -0,0 +1,541 @@
+.. _pipelines:
+
+===============================
+MLflow Pipelines (experimental)
+===============================
+
+MLflow Pipelines is an opinionated framework for structuring MLOps workflows that simplifies and
+standardizes machine learning application development and productionization. MLflow Pipelines
+makes it easy for data scientists to follow best practices for creating production-ready ML
+deliverables, allowing them to focus on utilizing their expert knowledge to develop excellent
+models. MLflow Pipelines also enables ML engineers and DevOps teams to seamlessly deploy these
```

Comment:
```suggestion
deliverables, allowing them to focus on developing excellent
models. MLflow Pipelines also enables ML engineers and DevOps teams to seamlessly deploy these
```

Reply:
Not sure if the prior was relevant

Reply:
Agreed, it wasn't. Thanks for the simplification!